docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -2138,13 +2142,58 @@ public class CustomGlobalFilter implements GlobalFilter, Ordered {
 ----
 ====
 
+=== The Gateway Metrics Filter
+
+To enable gateway metrics, add spring-boot-starter-actuator as a project dependency. Then, by default, the gateway metrics filter runs as long as the property `spring.cloud.gateway.metrics.enabled` is not set to `false`. This filter adds a timer metric named `spring.cloud.gateway.requests` with the following tags:
+
+* `routeId`: The route ID.
+* `routeUri`: The URI to which the API is routed.
+* `outcome`: The outcome, as classified by link:https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/http/HttpStatus.Series.html[HttpStatus.Series].
+* `status`: The HTTP status of the request returned to the client.
+* `httpStatusCode`: The HTTP Status of the request returned to the client.
+* `httpMethod`: The HTTP method used for the request.
+
+In addition, through the property `spring.cloud.gateway.metrics.tags.path.enabled` (by default, set to false), you can activate an extra metric with the tag:
+
+* `path`: Path of the request.
+
+These metrics are then available to be scraped from `/actuator/metrics/spring.cloud.gateway.requests` and can be easily integrated with Prometheus to create a link:images/gateway-grafana-dashboard.jpeg[Grafana] link:gateway-grafana-dashboard.json[dashboard].
+
+NOTE: To enable the prometheus endpoint, add `micrometer-registry-prometheus` as a project dependency.
+
+[[local-cache-response-global-filter]]
+=== The Local Response Cache Filter
+
+The `LocalResponseCache` runs if its associated property is enabled `spring.cloud.gateway.filter.local-response-cache.enabled` and will activate a local cache for all responses that meet the following criteria:
+- The request is a bodyless GET
+- The response has one of the following status codes: HTTP 200 (OK), HTTP 206 (Partial Content) or HTTP 301 (Moved Permanently)
+- HTTP `Cache-Control` header allows caching (that means it doesn't have any of the following values: `no-store` present in the request, `no-store` or `private` present in the response)
+
+It accepts two configuration parameters: `spring.cloud.gateway.filter.local-response-cache.size` to set the maximum size of the cache to evict entries for this route, it takes size format in KB, MB and GB; and `spring.cloud.gateway.filter.local-response-cache.timeToLive` to set time to expire a cache entry expressed in s for seconds, m for minutes and h for hours.
+If none of these parameters are configured but the global filter is enabled, by default it configures 5 minutes of time to live for the cached response.
+
+This filter also implements the automatic calculation of the `max-age` value in the HTTP `Cache-Control` header, if `max-age` is present on the original response the value will be rewritten with the number of seconds set in the `timeToLive` configuration parameter; and in consecutive calls this value will be recalculated with the number of seconds left until the response expires.
```

Comment:
Change to:

This filter also implements the automatic calculation of the `max-age` value in the HTTP `Cache-Control` header. If `max-age` is present on the original response, the value is rewritten with the number of seconds set in the `timeToLive` configuration parameter. In subsequent calls, this value is recalculated with the number of seconds left until the response expires.

(I would put each sentence on its own line, too.)